### ISSUE TITLE
Failed installation new Kennisnet organisation for testing

### ISSUE THREAD

**Issue by paulvandenbraken** [minvws/nl-kat-coordination#3479]
_Please add `bug`, the name of any relevant modules (e.g. `rocky`), and any other relevant labels to your issue._

**Describe the bug**
A newly entered organizational URL does not lead to software Instances

**To Reproduce**
- Create new organization <done>
- Objects and boefjes <done>
  - WebAnalysis boefje: on
  - Wepalyzer normalizer on
  - Snyk.io normalizer on
- No SW instances are then created.

**Expected behavior**
A new organisation for periodic scanning

**Screenshots**
- - -

**OpenKAT version**
1.16

**Desktop (please complete the following information if relevant):**
- - -

**Additional context**
- - -

**Comment by stephanie0x00** [comment]
@paulvandenbraken  could you confirm you have performed the steps below as well? 

- The boefjes `nmap tcp`, `dns records` and `dns zone` are enabled. 
- The host/URL you added has L2 clearance or above. 

The mentioned boefjes are required before the correct object is created in the backend that can be picked up by Webpage Analysis. This is currently not very clear in the interface and the UX/UI team is aware of this issue.

**Comment by underdarknl** [comment]
related:
https://github.com/minvws/nl-kat-coordination/issues/3037

**Comment by stephanie0x00** [comment]
Yes, we debugged the issue right away. It  is caused by #3487, which appeared to be a result of #3037.
